### ISSUE TITLE
ka_12 values incorrect?

### ISSUE THREAD

**Issue by msaimper** [Velleman/KA12#1]
Hello all,

not sure if it happened to other people as well but in order to get the correct redirection of the channels I had to modify the `ka12_values` array defined here:

https://github.com/Velleman/KA12/blob/master/Velleman_KA12/Velleman_KA12.cpp#L5

in this way:

original one documented in this repo: 
{180,182,181,183,177,176,179,178,173,175,174,171,168,169,154,153,152,155,157,159,156,158,170,172}

the one which redirects all channels correctly for me (1-24): 
{157,159,156,158,170,172,180,182,181,183,177,176,179,178,173,175,174,171,168,169,154,153,152,155}

so basically the block "157,159,156,158,170,172" had to be moved at the beginning ; the rest is unchanged.
weird right?

Cheers
